.github/workflows/end2end.yaml

```diff
@@ -556,8 +553,9 @@ jobs:
     needs: [build-kafka, build-test-image]
     runs-on:
       - ubuntu-22.04-8core
-      # Enable this for Ring-based tests
-      # - scality-cloud
+    env:
+      ENABLE_RING_TESTS: "true"
+      GIT_ACCESS_TOKEN: ${{ secrets.GIT_ACCESS_TOKEN }}
```

Comment:
credentials should not be added to env, esp. at such broad "scale" : best to pass them via inputs, in the place where they are needed